packages/animations/browser/src/dsl/animation_ast_builder.ts

```diff
@@ -513,7 +514,8 @@ export class AnimationAstBuilderContext {
   public currentQuerySelector: string|null = null;
   public currentAnimateTimings: TimingAst|null = null;
   public currentTime: number = 0;
-  public collectedStyles: {[selectorName: string]: {[propName: string]: StyleTimeTuple}} = {};
+  public collectedStyles: Map<string, Map<string, StyleTimeTuple>> =
```

Comment:
We can drop type annotation here - it'd be inferred.
```suggestion
  public collectedStyles =
```